### ISSUE TITLE
aws_cur_report_definition doesn't support Parquet

### ISSUE THREAD

**Issue by waffleshop** [terraform-providers/terraform-provider-aws#8765]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

0.12.0

### Affected Resource(s)

* aws_cur_report_definition

### Terraform Configuration Files

```hcl
provider "aws" {
  profile     = "default"
  region      = "us-east-1"
  version     = "2.12.0"
}

resource "aws_cur_report_definition" "cost_report" {
  report_name                = "mycostreport"
  time_unit                  = "HOURLY"
  format                     = "Parquet"
  compression                = "Parquet"
  additional_schema_elements = ["RESOURCES"]
  s3_bucket                  = "mybucketname"
  s3_prefix                  = "hourly-costs"
  s3_region                  = "us-east-1"
  additional_artifacts       = ["ATHENA"]
}
```

### Expected Behavior

Resource should have been created. This AWS API supports these values according to their [documentation](https://docs.aws.amazon.com/aws-cost-management/latest/APIReference/API_cur_ReportDefinition.html). Additionally, this AWS resource can be created using the Python boto3 package.

### Actual Behavior

`terraform plan` throws the following errors:

```
Error: expected format to be one of [textORcsv], got Parquet
...
Error: expected compression to be one of [GZIP ZIP], got Parquet
...
Error: expected additional_artifacts.0 to be one of [QUICKSIGHT REDSHIFT], got ATHENA
...
```

### Steps to Reproduce

1. `terraform apply`

### References

* https://docs.aws.amazon.com/aws-cost-management/latest/APIReference/API_cur_ReportDefinition.html

**Comment by zioalex** [comment]
Can we have also add support for ATHENA is this issue or would be better to have a new on?.

**Comment by robbruce** [comment]
@zioalex see https://github.com/terraform-providers/terraform-provider-aws/pull/12428

**Comment by bflad** [comment]
Support for this functionality has been merged and will release with version 3.5.0 of the Terraform AWS Provider, likely tomorrow. Thanks to @robbruce for the implementation. 👍
